lxd/images.go

```diff
@@ -3548,7 +3548,7 @@ func imageAliasesGet(d *Daemon, r *http.Request) response.Response {
 	var responseStr []string
 	var responseMap []api.ImageAliasesEntry
 	err = s.DB.Cluster.Transaction(r.Context(), func(ctx context.Context, tx *db.ClusterTx) error {
-		names, err := tx.GetImageAliases(ctx, projectName)
```

Comment:
After this, I believe `GetImageAliases` is left unused, so you can just update `GetImageAliases` to also return the aliases' image types instead of creating a new function `GetImageAliasesTypes`